jsonexample/lib/json_strings.dart

```diff
@@ -3,22 +3,22 @@
 // found in the LICENSE file.
 
 abstract class JsonStrings {
-  static final String listOfInts = '[1, 2, 3]';
+  static const String listOfInts = '[1, 2, 3]';
```

Comment:
nit: Could probably remove the `String` type annotations on these. This is some of the first Flutter code I ever wrote, which shows sometimes. 😄

Reply:
done